bcda/cclf/cclf_test.go

```diff
@@ -243,6 +243,18 @@ func (s *CCLFTestSuite) TestOrderACOs() {
 	assert.Equal(s.T(), "A8765", acoOrder[1])
 	assert.Regexp(s.T(), "A1111|A0246", acoOrder[2])
 	assert.Regexp(s.T(), "A1111|A0246", acoOrder[3])
+
+    // Check for no duplicates
+    var dupcheck = make(map[string]bool, 4)
+    for _, v := range acoOrder {
```

Comment:
This test will pass as-is, right? Meaning if you didn't have the utils.Dedup() [here](https://github.com/CMSgov/bcda-app/pull/625/files#diff-59e95aaede55a09c483036234c8a21d5210e6fa6a0948a26a8c0b90fda49ca8fR403), this test would be pass with this additional check?

Can we update the unit-test-db data to include a duplicate ACO entry? To do that, I think we have to create another entry in the systems table that references the same group_id.


Example:
```
bcda_test=# INSERT INTO systems (group_id, deleted_at) VALUES ('dbbd1ce1-ae24-435c-807d-ed45953077ff', NULL) RETURNING id;
 id
----
  9
(1 row)

INSERT 0 1
bcda_test=# INSERT INTO secrets (system_id) VALUES (9);
INSERT 0 1
```
See this README entry for details on updating the unit-test-db: https://github.com/CMSgov/bcda-app/blob/master/README.md#updating-seed-data-for-unit-tests

Reply:
Yep

Reply:
A duplicate "A4321" added to group table.  When the Dedup function is commented out... the following test fails.
<img width="853" alt="Screen Shot 2021-01-27 at 6 31 43 PM" src="https://user-images.githubusercontent.com/14236737/106068099-00548280-60ce-11eb-96f4-acbeaf0388c2.png">